File: src/main/java/com/sscanner/team/user/responseDto/UserPhoneUpdateResponseDto.java
package com.sscanner.team.user.responseDto;

import com.sscanner.team.User;

public record UserPhoneUpdateResponseDto(
        String userId,
        String newPhone
) {
    public static UserPhoneUpdateResponseDto from(User user) {
        return new UserPhoneUpdateResponseDto(
                user.getUserId(),
                user.getPhone()
        );
    }
}

File: src/main/java/com/sscanner/team/user/responsedto/UserPhoneUpdateResponseDto.kt
package com.sscanner.team.user.responsedto

import com.sscanner.team.user.entity.User

data class UserPhoneUpdateResponseDto(
    val userId: String?,
    val newPhone: String
) {
    companion object {
        fun from(user: User): UserPhoneUpdateResponseDto {
            return UserPhoneUpdateResponseDto(
                userId = user.userId,
                newPhone = user.phone
            )
        }
    }
}